collections/_release_4_5/entry-platforms-apple-export-to-visionos.md

```diff
@@ -0,0 +1,26 @@
+---
+type: entry
+section: platforms
+subsection: apple
+rank: 1
+importance: 3
+anchor: export-to-visionos
+title: Export to visionOS
+text: |
+  You can now export for Apple’s XR platform, making Godot projects compatible with the Apple Vision Pro.
+
+  Godot 4.5 marks the first step of visionOS’ native integration. Currently, it is only possible to export a project as an "windowed app". Your game will appear as a window, floating in the 3D space of the user.
+
+  Fully immersive experience support is expected. We’ll keep you in the loop.
+
+  It goes without saying, but this is kinda a big deal: this is the first new platform supported natively by Godot since the project was open-sourced!
+contributors:
+  - name: Ricardo Sanchez-Saez
```

Comment:
I think it'd be worth noting somewhere in this block that Ricardo works on Apple's visionOS engineering team – in other words, this addition to Godot was done by Apple itself, which is pretty huge (unless I'm misunderstanding something)!

Reply:
I think it is what we should do! Ricardo did this work so he should be credited and Apple is who pays his salary, so there isn't anything to be worried about mentioning their involvement 👍

Reply:
Done!